README.md

```diff
@@ -125,32 +92,88 @@ If configured, the action supports setting the Jira issues's status when the Bug
     - mapping [str, str]
     - If defined, map the Bugzilla bug resolution to Jira issue resolution
 
-Example configuration:
+Minimal configuration:
 ```yaml
     whiteboard_tag: example
     contact: example@allizom.com
-    description: example configuration
-    enabled: true
-    module: jbi.actions.default_with_assignee_and_status
+    description: minimal configuration
     parameters:
       jira_project_key: EXMPL
-      status_map:
-        NEW: "In Progress"
-        FIXED: "Closed"
-      resolution_map:
-        FIXED: "Done"
-        DUPLICATE: "Duplicate"
-        WONTFIX: "Won't Do"
+```
+
+Full configuration, that will set assignee, change the Jira issue status and resolution.
+
+```yaml
+- whiteboard_tag: fidefe
+  contact: example@allizom.com
+  description: full configuration
+  enabled: true
+  module: jbi.actions.default
+  parameters:
+    jira_project_key: FIDEFE
+    steps:
```

Comment:
It appears that this "steps" section splits based on the event-type (which doesn't seem to be mentioned); adding these event types in the documentation might be helpful.

Another concern is, during the "update" event there is a possibility of creating a jira issue; based on the step name "update_issue" this implies an issue would be already present. Changing the name to "upsert" instead of "update" might work if the create functionality is still there. 


---

However, I question the value of separating these steps by the "events" (create, update, comment, attachment). 

I think each step should validate that if it should/n't run: 
- create_issue runs on both "update" and "create" events, only create a Jira ticket if this was already done (when "comment" events occur, do nothing)

---

Just some minor thoughts